pymc3/distributions/continuous.py

```diff
@@ -1410,3 +1410,45 @@ def logp(self, value):
             + (-tau * (value - mu)**2
             + tt.log(tau / np.pi / 2.)) / 2.,
             tau > 0, sd > 0)
+
+
+class Triangular(Continuous):
+    """
```

Comment:
Doc-string could be improved, see the other doc-strings that include the support of the parameters as well a latex string of the math.